.github/workflows/version_bump.yml

```diff
@@ -0,0 +1,31 @@
+name: Version Bump
+
+on:
+  create: # It will trigger whenever a new tag or branch is created
```

Comment:
This will probably be triggered by a manual workflow instead, pending https://github.com/interledger/rafiki/pull/1976#issuecomment-1757881005